packages/gatsby-plugin-emotion/package.json

```
@@ -26,7 +26,7 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.9.6",
```

Needs to be 7.0.0, else this would be breaking
```suggestion
    "@babel/core": "^7.0.0",
```